api/api-doc/storage_service.json

```diff
@@ -2574,6 +2574,70 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/tablets/add_replica",
+         "operations":[
+            {
+               "nickname":"add_tablet_replica",
+               "method":"POST",
+               "summary":"Adds replica to tablet",
+               "type":"void",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"ks",
+                     "description":"Keyspace name",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"table",
+                     "description":"Table name",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"token",
+                     "description":"Token owned by the tablet to add replica to",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"integer",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"dst_host",
+                     "description":"Destination host id",
```

Comment:
We have a similar description issue here. @xemul .

Reply:
Similar to what? Del-replica? Why doesn't "Destination host id" applies to add-replica?

Reply:
I meant  https://github.com/scylladb/scylladb/pull/18203/files#diff-7771677331ece83ba219b8ed9f7625afdae1f9fc4738d44c532c623c13f5eb13R2671. 

The word destination is copied from the move tablet api description which has both source and destination.

You changed remove_replica api: 

"Host id to remove replica from"

We can change the add_replica description too: 

"Host id / shard id  to add replica to"

It is clear than "Destination host id".

Reply:
OK, I'll include it in the followup pr